articles/synapse-analytics/sql/on-demand-workspace-overview.md

```diff
@@ -33,7 +35,7 @@ If you need to explore data in the data lake, gain insights from it or optimize
 
 - Basic discovery and exploration - Quickly reason about the data in various formats (Parquet, CSV, JSON) in your data lake, so you can plan how to extract insights from it.
 - Logical data warehouse – Provide a relational abstraction on top of raw or disparate data without relocating and transforming data, allowing always up-to-date view of your data. Learn more about [creating logical data warehouse](tutorial-logical-data-warehouse.md).
-- Data transformation - Simple, scalable, and performant way to transform data in the lake using T-SQL, so it can be fed to BI and other tools, or loaded into a relational data store (Synapse SQL databases, Azure SQL Database, etc.).
+- Data transformation - Simple, scalable, and performant way to transform data in the lake using T-SQL, so it can be fed to BI and other tools, or loaded into a relational data store (Synapse SQL databases, Azure SQL Database, etc.). BI tools like Power BI or Azure Analysis Service can connect programmatically to integration tools like Synapse Pipelines or Azure Data Factory and popular languages ​​like C# or Python.
```

Comment:
Added sentence seems doesn't seem like a fit to an article describing serverless sql pool, rather explains capability of BI tools connecting to integration tools.